### PR TITLE
Share to Snapchat

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
https://user-images.githubusercontent.com/36916764/209911772-e11cd81a-c433-4f72-a314-3cf4184e5223.MP4

Remaining steps:
-Design QA
-AJ/tech QA
-Update API key to prod Snapchat key once we get one (app is submitted for approval)

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/ios/AudiusReactNative/Info.plist [hunk 1]
```diff
@@ -42,7 +42,13 @@
 			<string>instagram-stories</string>
 			<string>instagram</string>
 			<string>itms-apps</string>
+      <string>snapchat</string>
 		</array>
+    <key>SCSDKClientId</key>
+    <string>6f90a744-e5d0-4115-ad14-fb1a356085cd</string>
```
Comment (sliptype) [inline packages/mobile/ios/AudiusReactNative/Info.plist:48]: Is this supposed to be secret? Or is it safe to have in here

File: packages/mobile/src/types/analytics.ts [hunk 2]
```diff
@@ -7,7 +7,11 @@ enum MobileEventNames {
   SHARE_TO_IG_STORY = 'Share to Instagram story - start',
   SHARE_TO_IG_STORY_CANCELLED = 'Share to Instagram story - cancelled',
   SHARE_TO_IG_STORY_ERROR = 'Share to Instagram story - error',
-  SHARE_TO_IG_STORY_SUCCESS = 'Share to Instagram story - success'
```
Comment (Kyle-Shanks) [inline packages/mobile/src/types/analytics.ts:10]: ah interesting. I just added some events to track rate CTA stuff in common. Is there a difference adding the events here?

**Review by sliptype — APPROVED** [review]
Sooooo sick! And we get the link too?! 🔥

**Review by Kyle-Shanks — APPROVED** [review]

File: packages/mobile/src/types/analytics.ts (hunk 2 above)
Comment (sliptype) [reply packages/mobile/src/types/analytics.ts:10]: Oh yeah these are mobile specific events which would have made sense for yours too @Kyle-Shanks

**Comment by nicoback2** [comment]
> Sooooo sick! And we get the link too?! 🔥

Yesss their api is way better

File: packages/mobile/src/types/analytics.ts (hunk 2 above)
Comment (Kyle-Shanks) [reply packages/mobile/src/types/analytics.ts:10]: gotcha. is it worth moving the review events here?

Comment (sliptype) [reply packages/mobile/src/types/analytics.ts:10]: Nah probably not

File: packages/mobile/ios/AudiusReactNative/Info.plist (hunk 1 above)
Comment (nicoback2) [reply packages/mobile/ios/AudiusReactNative/Info.plist:48]: not secret!

**Pull request merged**